mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSModelImportAsNode.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.mpsadapters
+
+import org.jetbrains.mps.openapi.model.SModel
+import org.modelix.model.api.BuiltinLanguages
+import org.modelix.model.api.IChildLink
+import org.modelix.model.api.IConcept
+import org.modelix.model.api.IConceptReference
+import org.modelix.model.api.IDeprecatedNodeDefaults
+import org.modelix.model.api.INode
+import org.modelix.model.api.INodeReference
+import org.modelix.model.api.IProperty
+import org.modelix.model.api.IReferenceLink
+import org.modelix.model.area.IArea
+
+class MPSModelImportAsNode(val importedModel: SModel, val importingModel: SModel) : IDeprecatedNodeDefaults {
+    override fun getArea(): IArea =
+        MPSArea(importingModel.repository)
+
+    override val isValid: Boolean
+        get() = TODO("Not yet implemented")
+    override val reference: INodeReference
+        get() = TODO()
```

Comment:
```suggestion
        get() = TODO("Not yet implemented")
    override val reference: INodeReference
        get() = TODO("Not yet implemented")
```

Let's be consistent here